android/app/src/main/java/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -258,7 +258,7 @@ private void initSearchPrivacyPrefsCallbacks()
     ((TwoStatePreference) pref).setChecked(isHistoryEnabled);
     pref.setOnPreferenceChangeListener((preference, newValue) -> {
       boolean newVal = (Boolean) newValue;
```

Comment:
```suggestion
    pref.setOnPreferenceChangeListener((preference, Boolean newValue) -> {
```

Reply:
If we remove (newVal) variable then we need to refactor it everywhere because it has been used later also and everywhere we need to type cast it to boolean will it be ok?

I am committing it please review it.